src/games/strategy/engine/framework/startup/launcher/LocalLauncher.java

```diff
@@ -49,20 +49,8 @@ protected void launchInNewThread(final Component parent) {
         game.setRandomSource(new ScriptedRandomSource());
       }
       m_gameData.getGameLoader().startGame(game, gamePlayers, m_headless);
-    } catch (final IllegalStateException e) {
+    } catch( IllegalStateException e ) {
       exceptionLoadingGame = e;
-      Throwable error = e;
-      while (error.getMessage() == null) {
-        error = error.getCause();
-      }
-      final String message = error.getMessage();
-      m_gameLoadingWindow.doneWait();
-      SwingUtilities.invokeLater(new Runnable() {
-        @Override
-        public void run() {
-          JOptionPane.showMessageDialog(null, message, "Warning", JOptionPane.WARNING_MESSAGE);
-        }
-      });
```

Comment:
So wouldn't it be best to still leave the old exception handling here as well in case it is hit in other situations. So first catch MapNotFoundException then catch the IllegalStateException?